packages/documentation/src/pages/platform/front-end-standards/documented-decisions/summary.mdx

```diff
@@ -13,3 +13,4 @@ title: Summary of documented decisions
 - March 2018 - [Why we forked react-jsonschema-form](/platform/front-end-standards/documented-decisions/rjsf-fork)
 - November 2018 - [Why we omit the signin from Teamsite pages](/platform/front-end-standards/documented-decisions/teamsite-signin)
 - December 2018 - [Why we use Mocha instead of Jest](/platform/front-end-standards/documented-decisions/jest)
+- December 2019 - [Web-Fonts] (/platform/front-end-standards/documented-decisions/web-fonts)
```

Comment:
mention web performance in the title e.g. `Reducing site performance impact of web fonts`

Reply:
ok